vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectInformationLayout.java

```diff
@@ -0,0 +1,75 @@
+package life.qbic.datamanager.views.project.create;
+
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.formlayout.FormLayout.ResponsiveStep;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import java.util.Objects;
+import life.qbic.datamanager.views.components.CardLayout;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * <b>short description</b>
+ *
+ * <p>detailed description</p>
+ *
+ * @since <version tag>
+ */
+@SpringComponent
+public class ProjectInformationLayout extends Composite<CardLayout> {
+
+  final TextField titleField = new TextField();
+  final Button saveButton = new Button("Save");
+  final Button cancelButton = new Button("Cancel");
```

Comment:
Title Field and Buttons should be on the outer layout.
Bzw. the title 'Create Project' should be and this card will have a different title (Project Information)